ml-algorithms/src/main/java/org/opensearch/ml/engine/memory/ConversationMessage.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.ml.engine.memory;
+
+import lombok.Builder;
+import lombok.Getter;
+import lombok.Setter;
+
+public class ConversationMessage extends BaseMessage {
```

Comment:
Seems this class not being used in feature branch. Remove this class?

Reply:
removed